wix-embedded-mysql/src/main/java/com/wix/mysql/MysqlClient.java

```diff
@@ -20,41 +23,60 @@
 
     private final MysqldConfig config;
     private final MysqldExecutable executable;
+    private final String schemaName;
 
-    public MysqlClient(
-            final MysqldConfig config,
-            final MysqldExecutable executable) {
+    public MysqlClient(final MysqldConfig config, final MysqldExecutable executable) {
+        this(config, executable, SystemDefaults.SCHEMA);
+    }
 
+    public MysqlClient(final MysqldConfig config, final MysqldExecutable executable, final String schemaName) {
         this.config = config;
         this.executable = executable;
+        this.schemaName = schemaName;
+    }
+
+    public void executeScripts(final List<File> files) {
+        for (File file: files) {
+            execute(format("source %s", file.getAbsolutePath()));
+        }
+    }
+
+    public void executeScripts(final File... files) {
+        for (File file: files) {
+            execute(format("source %s", file.getAbsolutePath()));
+        }
     }
 
-    public void apply(String sqlStatement) {
-        String statement = String.format((Platform.detect() == Windows) ? "\"%s\"" : "%s", sqlStatement);
+    public void executeCommands(final String... sqls) {
+        for (String sql: sqls) {
+            execute(sql);
+        }
+    }
 
+    private void execute(final String sql) {
+        String command = (Platform.detect() == Windows) ? format("\"%s\"", sql): sql;
```

Comment:
It's not an excuse - it's motivation.

Otherwise I don't have a good reason for one or another. I will see if I can migrate all usage for mysqld executable to jdbc - in this case I agree that less native calls is better.